lib/archethic/transaction_chain.ex

```diff
@@ -994,4 +1036,24 @@ defmodule Archethic.TransactionChain do
         {:error, :network_issue}
     end
   end
+
+  @doc """
+  Retrieve the First transaction address for a chain from P2P Quorom
+  """
+  @spec fetch_first_transaction_address_remotely(address :: binary()) ::
+          {:ok, binary()} | {:error, :network_issue}
+  def fetch_first_transaction_address_remotely(address) when is_binary(address) do
```

Comment:
This function should take the node list in parameter, as TransactionChain may not know the context where the function is called.

Reply:
?

Reply:
All the function for fetching data have the node list in parameter, the charge to get the storage node of something is the reponsability of the function caller , beause the list of nodes may not be the current authorized and available node. In the self repair for example